extensions/scarb-doc/src/types.rs

```diff
@@ -23,12 +23,14 @@ use crate::db::ScarbDocDatabase;
 #[derive(Serialize, Clone)]
 pub struct Crate {
     pub root_module: Module,
+    pub name: String,
 }
 
 impl Crate {
-    pub fn new(db: &ScarbDocDatabase, crate_id: CrateId) -> Self {
+    pub fn new(db: &ScarbDocDatabase, crate_id: CrateId, name: String) -> Self {
         Self {
             root_module: Module::new(db, ModuleId::CrateRoot(crate_id)),
+            name,
```

Comment:
can't you extract this info from database given you know `CrateId`?

Reply:
Done